client/src/components/ReviewAndRating/StarRating.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.span`
+  display: flex;
+  align-items: flex-start;
+`;
+
+const AveRating = styled.span`
+  font-size: 60px;
+  margin-right: 20px;
+
+`;
+
+const StarOuter = styled.span`
+  & {
+    position: relative;
+    display: inline-block;
+  }
+
+  &:before {
+    content: "★★★★★";
+    color: #ebebeb;
+  }
+`;
+
+const StarInner = styled.span`
+  & {
+    position: absolute;
+    top: 0;
+    left: 0;
+    overflow: hidden;
+    width: ${props => props.starWidth}%;
```

Comment:
👏

Reply:
Feel free to use this component in your widget!